### PR TITLE
Deploy to staging and prod via GitHub

### PR DESCRIPTION
**Pull request by frankieroberto**
This switches our continuous deployment to happen from GitHub Actions instead of Travis CI.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        echo 'deb https://packages.cloudfoundry.org/debian stable main' | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+        sudo apt-get -qq update
+        sudo apt-get install cf7-cli
+    - name: Deploy to staging
+      env:
+        SPACE: staging
+        APP_NAME: cosmetics-web
+        SUBMIT_HOST: staging-submit.cosmetic-product-notifications.service.gov.uk
+        SEARCH_HOST: staging-search.cosmetic-product-notifications.service.gov.uk
+        CF_USERNAME: ${{ secrets.PaaSUsernameInt }}
```
Comment (slawosz) [inline .github/workflows/deploy.yml]: You are using `int` secrets, should be `secrets.PaaSUsernameStaging`

File: cosmetics-web/deploy.sh [hunk 2]
```diff
@@ -1,95 +1,32 @@
...
+then
+  echo "Please set your submit host, eg SUBMIT_HOST=submit.cosmetic-product-notifications.service.gov.uk"
+  exit
 fi
-APP=cosmetics-web
-APP_PREEXISTS=$(cf app $APP && echo 0 || echo 1)
 
-if [[ $APP_PREEXISTS ]]; then
-    # We should deploy to a temporary location such that we can do a blue-green deployment
-    NEW_SUBMIT_HOSTNAME=$SUBMIT_HOSTNAME-temp
```
Comment (slawosz) [inline cosmetics-web/deploy.sh]: Please make sure that `cf7` provides `blue-green` deployment.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (frankieroberto) [reply .github/workflows/deploy.yml]: 👍

File: cosmetics-web/deploy.sh (hunk 2 above)
Comment (frankieroberto) [reply cosmetics-web/deploy.sh]: Seems to work!

**Pull request merged**